wizard/wizard_log_reply_with_attachment.py

```diff
@@ -22,31 +30,48 @@ class WizardLogReplyAttachment(osv.osv_memory):
         'info': lambda *a: _('Select a file to send as an attachment')
     }
 
+
     def log_reply(self, cursor, uid, ids, context=None):
         if context is None:
             context = {}
         case_obj = self.pool.get('crm.case')
+        conf_obj = self.pool.get('res.config')
         attachment_obj = self.pool.get('ir.attachment')
         wiz = self.browse(cursor, uid, ids[0], context=context)
 
         ctx = context.copy()
         attachment_ids = []
-        if wiz.file:
-            attachment_data = {
-                'name': wiz.filename,
-                'content': wiz.file
-            }
-            att_id = attachment_obj.create(cursor, uid, {
-                'description': _(
-                    "From CRM case [{}] {}"
-                ).format(wiz.case_id.id, wiz.case_id.name),
-                'datas_fname': attachment_data['name'],
-                'name': attachment_data['name'],
-                'datas': attachment_data['content'],
-                'res_model': 'crm.case',
-                'res_id': wiz.case_id.id
-            })
-            attachment_ids.append(att_id)
+        total_size = 0
+        max_mb_size = float(conf_obj.get(
+            cursor, uid, 'crm_poweremail_max_size_mb', 20))
+        for x in xrange(1, 6):
```

Comment:
```suggestion
        for x in range(1, 6):
```

Use `range` instead of `xrange` for better py3 compatibility